docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,328 @@
+    "\n",
+    "This guide will show you how to finetune a multilingual CLIP model for a text to image retrieval in non-English languages.\n",
+    "\n",
+    "*Note, please consider switching to GPU/TPU Runtime for faster inference.*\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed1e7d55-a458-4dfd-8f4c-eeb02521c221",
+   "metadata": {},
+   "source": [
+    "## Install"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9261d0a7-ad6d-461f-bdf7-54e9804cc45d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install 'finetuner[full]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11f13ad8-e0a7-4ba6-b52b-f85dd221db0f",
+   "metadata": {},
+   "source": [
+    "## Task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed1f88d4-f140-48d4-9d20-00e628c73e38",
+   "metadata": {},
+   "source": [
+    "We'll be finetuning multilingual CLIP on the `toloka-fashion` dataset, which contains information about fashion products, with all descriptions being in German.  \n",
+    "\n",
+    "Each product in the dataset contains several attributes, we will be making use of the image and category attributes to create a [`Document`](https://docarray.jina.ai/fundamentals/document/#document) containing two [chunks](https://docarray.jina.ai/fundamentals/document/nested/#nested-structure), one containing the image and another containing the category of the product."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a40f0b1-7272-4ae6-9d0a-f5c8d6d534d8",
+   "metadata": {},
+   "source": [
+    "## Data\n",
+    "We will use the `toloka-fashion` dataset, which we have already pre-processed and made available on the Jina AI Cloud. You can access it by like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4420a4ac-531a-4db3-af75-ebb58d8f828b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import finetuner\n",
+    "from docarray import DocumentArray, Document\n",
+    "\n",
+    "finetuner.login(force=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bab5c3fb-ee75-4818-bd18-23c7a5983e1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data = DocumentArray.pull('toloka-fashion-train-data', show_progress=True)\n",
+    "eval_data = DocumentArray.pull('toloka-fashion-eval-data', show_progress=True)\n",
+    "\n",
+    "train_data.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b859e9c-99e0-484b-98d5-643ad51de8f0",
+   "metadata": {},
+   "source": [
+    "## Backbone Model\n",
+    "Currently, we only support one multilingual CLIP model, which has been made available by [open-clip](https://github.com/mlfoundations/open_clip). This model is the `xlm-roberta-base-ViT-B-32`, which has been trained on the `laion5b` dataset."
```

Comment:
"Currently, we only support one multilingual CLIP model. This model is the `xlm-roberta-base-ViT-B-32` from [open-clip](https://github.com/mlfoundations/open_clip), which has been trained on the [`laion5b` dataset](https://github.com/LAION-AI/laion5B-paper)."